shapeworks_cloud/core/rest.py

```diff
@@ -16,6 +18,16 @@
 from . import filters, models, serializers
 from .tasks import groom, optimize
 
+DB_WRITE_ACCESS_LOG_FILE = Path('logging/db_write_access.log')
+if not os.path.exists('logging'):
+    os.mkdir('logging')
+
+
+def log_write_access(*args):
+    with open(DB_WRITE_ACCESS_LOG_FILE, 'w+') as log_file:
```

Comment:
Don't you want `a` mode

Reply:
Changed in [89627c1](https://github.com/girder/shapeworks-cloud/pull/319/commits/89627c1313f607f233d94e4dc10a665657c5b905)